Tiltfile

```diff
@@ -164,7 +162,8 @@ def enable_provider(name):
         context = context,
         dockerfile_contents = dockerfile_contents,
         target = "tilt",
-        entrypoint = "sh /start.sh /manager",
+        # TODO(vincepri): Fix this in Tilt, given that this breaks conversion webhooks.
+        # entrypoint = "sh /start.sh /manager",
```

Comment:
Any more information on how it breaks the conversion webhooks? Is it because we are deploying them as a single k8s_deployment instead of multiple?

Reply:
I was working with them to fix it, seems we might have a solution already

Reply:
Done! The trick was to make it an array, also this got fixed the other day https://github.com/windmilleng/tilt/issues/2918